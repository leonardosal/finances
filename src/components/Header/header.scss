.header {
  height: 50px;
  padding: 16px;
  background-color: #0ff;
  font-size: 1.5rem;
  display: flex;
  align-items: center;
}
