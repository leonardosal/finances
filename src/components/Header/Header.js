import React from 'react';

import './header.scss';

const Header = () => (
  <header className="header">
    <span>Finances</span>
  </header>
);

export default Header;
