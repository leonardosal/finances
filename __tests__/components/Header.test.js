import React from 'react';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';

import Header from '../../src/components/Header';

it('should render header component', () => {
  const component = mount(<Header />);
  expect(toJson(component)).toMatchSnapshot();
});
