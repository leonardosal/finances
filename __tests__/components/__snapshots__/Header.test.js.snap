// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render header component 1`] = `
<Header>
  <header
    className="header"
  >
    <span>
      Finances
    </span>
  </header>
</Header>
`;
